Remotion/Web/Development.WebTesting.IntegrationTests/IntegrationTest.cs

```diff
@@ -52,8 +54,12 @@ protected IDriver Driver
     [OneTimeSetUp]
     public void IntegrationTestOneTimeSetUp ()
     {
+      DriverConfigurationOverride driverConfigurationOverride = null;
+      if (Debugger.IsAttached || RequiresUserInterfaceAttribute.GetPropertyValueFromAmbientTestContext())
```

Comment:
Seeing this used, it feels kind of strange. Wouldn't an extension method be cleaner. or just a method that gets the testcontext injected?

Reply:
I think it is fine as the attribute is a NUnit specific attribute. Separating the logic from the attribute would make it more difficult to find the related code.

Reply:
Yes, I can see how separating the types would be awkward (and why is c# so finicky about this...) What do you think about `RequiresUserInterfaceAttribute.GetPropertyValue(TestContext.Current)` ?